bin/rucio

```diff
@@ -2029,6 +2029,61 @@ def delete_metadata(args):
     raise NotImplementedError
 
 
+@exception_handler
+def get_generic_metadata(args):
```

Comment:
get_did_meta ? if it doesn't clash with another method

Reply:
yeah, did_meta sounds better (for all the other ones as well)